config.js

```diff
@@ -56,19 +62,19 @@ module.exports = {
     healthCheckUrl: process.env.NOTIFY_HEALTH_CHECK_URL || 'https://api.notifications.service.gov.uk/_status',
   },
   cmdApi: {
-    url: get('CMD_API_URL', 'http://localhost:9191'),
+    url: get('CMD_API_URL', 'http://localhost:8081'),
```

Comment:
I _think_ this port is needed for integration tests, but don't quote me on that one